src/sqlfluff/core/default_config.cfg

```diff
@@ -63,6 +63,7 @@ indented_joins = False
 indented_ctes = False
 indented_using_on = True
 indented_on_contents = True
+allow_implicit_indents = False
```

Comment:
Did you mean to include this line? Seems unrelated to coverage checks.

Reply:
oops - no I didn't but it's not harmful 🤷